erikiado/jp2_online
# Task/fill family form

Note: Remember to create a new Escuela in order to be able to test this manually.
An issue will be created for the creation a database initialization file.

#### What's this PR do?
It enables a user to create a new study, as well as updating the family, adding members, and updating them.

#### Where should the reviewer start?
The reviewer should start on the captura/views.py

The first thing to check is the creation of a new study
Then updating a family
Then creation of an integrante
Then editing of an integrante

#### How should this be manually tested?
You should login as a capturista, and create a new study, fill the information as it is being asked, and then create a family member of each type.

#### Any background context you want to provide?



This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
